### PR TITLE
added a controller for generating qrcodes

### PR DESCRIPTION
**Pull request by jaykayone**
controller for generating qrcodes for geoportail urls

### CODE REVIEW

**Comment by jaykayone** [comment]
Closes #211

**Comment by sbrunner** [comment]
Looks good for me :-)

File: geoportailv3/views/qr.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+                '^http[s]{0,1}://.*[g-o|geoportal|geoportail]{1}.lu.*$')
+
+    @view_config(route_name='qr')
+    def getqrcode(self):
+        url = self.request.params.get('url', '')
+        if len(url) == 0:
+            return HTTPBadRequest()
+        if not self.regex.match(url):
+            return HTTPNotAcceptable(
+                    "not a valid url for this grcode generator")
```
Comment (pgiraud) [inline geoportailv3/views/qr.py]: grcode -> qrcode

Comment (pgiraud) [inline geoportailv3/views/qr.py]: even better "QR code"

**Comment by elemoine** [comment]
@sbrunner where should the package version be set in this case? qrcode is geoportailv3-specific package dependency…

**Comment by sbrunner** [comment]
two possible way :
The easiest is to fix it in the `setup.py`.
Or create a `version.txt` file and add in the `geoportailv3.mk` file:
`REQUIREMENTS += -r version.txt` :-)

**Comment by elemoine** [comment]
+1 for fixing the version in `setup.py`. I don't see the advantage of having a `versions.txt` file here.

**Comment by jaykayone** [comment]
do I understand this correctly?
should I add ==5.1 in the setup.py ?

**Comment by elemoine** [comment]
@jaykayone yes!

**Comment by jaykayone** [comment]
Done, shouldn't this also be dont for elastisearch, python_ldap etc?

**Comment by jaykayone** [comment]
can this be merged?

**Comment by sbrunner** [comment]
For me yes :-)

**Comment by elemoine** [comment]
It looks like this broke the build. See https://travis-ci.org/Geoportail-Luxembourg/geoportailv3/builds/53660413.

**Comment by jaykayone** [comment]
where do you see a correlation to my merge? Travis was OK before I merged
I think the build is broken because of #327 and #328

**Comment by jaykayone** [comment]
OK I see the message now, sorry. 
No idea why it has a problem now but not when I merged ...

**Pull request merged**